<ui version="4.0" >
 <class>EditorCalltipsPage</class>
 <widget class="QWidget" name="EditorCalltipsPage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>369</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" >
   <item>
    <widget class="QLabel" name="headerLabel" >
     <property name="text" >
      <string>&lt;b>Configure Calltips&lt;/b></string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line18" >
     <property name="frameShape" >
      <enum>QFrame::HLine</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ctEnabledCheckBox" >
     <property name="toolTip" >
      <string>Select this to enable calltips</string>
     </property>
     <property name="text" >
      <string>Calltips Enabled</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="textLabel2" >
        <property name="text" >
         <string>Visible calltips:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="ctVisibleSlider" >
        <property name="toolTip" >
         <string>Move to set the maximum number of calltips shown (0 = all available)</string>
        </property>
        <property name="maximum" >
         <number>20</number>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickInterval" >
         <number>1</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="lCDNumber5" >
        <property name="toolTip" >
         <string>Displays the maximum number of calltips to be shown</string>
        </property>
        <property name="numDigits" >
         <number>2</number>
        </property>
        <property name="segmentStyle" >
         <enum>QLCDNumber::Flat</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="title" >
      <string>Colours</string>
     </property>
     <layout class="QHBoxLayout" >
      <item>
       <widget class="QLabel" name="TextLabel2_2_2_2" >
        <property name="text" >
         <string>Background colour:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="calltipsBackgroundButton" >
        <property name="minimumSize" >
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>Select the background colour for calltips.</string>
        </property>
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox" >
     <property name="title" >
      <string>Plug-In Behavior</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <widget class="QCheckBox" name="ctScintillaCheckBox" >
        <property name="toolTip" >
         <string>Select to show QScintilla provided calltips, if the selected plugin fails</string>
        </property>
        <property name="whatsThis" >
         <string>Qscintilla provided calltips are shown, if this option is enabled and calltips shall be provided by a plug-in (see calltips sub-page of the plug-in) and the plugin-in doesn't deliver any calltips.</string>
        </property>
        <property name="text" >
         <string>Show QScintilla calltips, if plug-in fails</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>388</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ctEnabledCheckBox</tabstop>
  <tabstop>ctVisibleSlider</tabstop>
  <tabstop>calltipsBackgroundButton</tabstop>
  <tabstop>ctScintillaCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>ctEnabledCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>68</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel" >
     <x>77</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ctEnabledCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>199</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel" >
     <x>199</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ctVisibleSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lCDNumber5</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>285</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel" >
     <x>372</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
